.travis.yml

```diff
@@ -13,12 +13,19 @@ python:
 stages:
   - lint
   - test
+  - name: deploy
+    if: tag is present
 
 matrix:
   include:
     - stage: lint
       name: flake8
       python: '3.7'
+    - stage: deploy
+      name: Deploy to PyPI
+      python: '3.7'
```

Comment:
Do we really need Python 3.7 for deploying? Just curious about why we don't use Python 2.7 ;)

Reply:
I suppose that any Python version could be used for deploying.
At the same time, I believe that it makes sense to use a version which [will be EOLed](https://devguide.python.org/#status-of-python-branches) later than sooner, so Python 2.7 with end-of-life on 2020-01-01 is not a good option.